exporter/awsemfexporter/grouped_metric.go

```diff
@@ -35,6 +35,13 @@ func addToGroupedMetric(pmd pmetric.Metric, groupedMetrics map[interface{}]*grou
 	}
 
 	for i := 0; i < dps.Len(); i++ {
+		// Drop stale or NaN metric values
+		if dps.IsStaleOrNaN(i) {
+			if config != nil && config.logger != nil {
+				config.logger.Debug("dropped metric with nan value", zap.String("metric.name", pmd.Name()))
```

Comment:
Should this include the metric attributes?  The name alone is not identifying.

Reply:
Is there a consistent way to log this without being too verbose? Or should we just log all metric attributes along with the name?

Reply:
I've updated the `IsStaleOrNaN` function signature to also return the attributes for each data point. I don't love this but it seemed like the cleanest way to ensure only one log line had to be added. I chose this over adding the logger as an function parameter to `IsStaleOrNaN`.